src/ray/gcs/actor_state_accessor.h

```diff
@@ -4,40 +4,35 @@
 #include "ray/common/id.h"
 #include "ray/gcs/callback.h"
 #include "ray/gcs/subscription_executor.h"
-#include "ray/gcs/tables.h"
 
 namespace ray {
 
 namespace gcs {
 
-class RedisGcsClient;
-
 /// \class ActorStateAccessor
-/// ActorStateAccessor class encapsulates the implementation details of
+/// ActorStateAccessor class introduce the interface of
 /// reading or writing or subscribing of actor's specification (immutable fields which
 /// determined at submission time, and mutable fields which are determined at runtime).
```

Comment:
What about this?
```
`ActorStateAccessor` is a sub-interface of `GcsClient`. This class includes all the methods that are related to accessing actor information in the GCS.
```
Reason: the details of "reading, writing and subscribing" can be found from concrete methods in this class, and they could change in the future. So I think, for the class's documentation, it's more important to explain the overall purpose of this class, and its relationship with other classes. 

Also, per this [doc](https://docs.google.com/document/d/1T3oing7ff3HC9iQZ4qZKowNXFGKeBhqKP4BEjqdN3vM/edit#heading=h.twqv2ptqtsi0), let's rename this class to `ActorInfoAccessor`?